INSTALL

```diff
@@ -152,145 +151,66 @@ We provide a `setup.cfg
-
-    Versions 8.6.0 and 8.6.1 are known to have issues that may result
-    in segfaults when closing multiple windows in the wrong order.
-
-:term:`pyqt` 4.4 or later
-    The Qt4 widgets library python wrappers for the Qt4Agg backend
-
-:term:`pygtk` 2.4 or later
-    The python wrappers for the GTK widgets library for use with the
-    GTK or GTKAgg backend
-
-:term:`wxpython` 2.8 or later
-    The python wrappers for the wx widgets library for use with the
-    WX or WXAgg backend
-
-Optional external programs
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-`ffmpeg <https://www.ffmpeg.org/>`__/`avconv <https://libav.org/avconv.html>`__ or `mencoder <http://www.mplayerhq.hu/design7/news.html>`__
-    Required for the animation module to be save out put to movie
-    formats.
-
-`ImageMagick <http://www.imagemagick.org/script/index.php>`__
-    Required for the animation module to be able to save to animated gif.
-
-Optional dependencies
-^^^^^^^^^^^^^^^^^^^^^
-
-`Pillow <http://python-pillow.org/>`__
-    If Pillow is installed, matplotlib can read and write a larger
-    selection of image file formats.
-
-`pkg-config <https://www.freedesktop.org/wiki/Software/pkg-config/>`__
-    A tool used to find required non-python libraries.  This is not strictly
-    required, but can make installation go more smoothly if the libraries and
-    headers are not in the expected locations.
-
-
-Required libraries that ship with matplotlib
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-:term:`agg` 2.4
-    The antigrain C++ rendering engine.  matplotlib links against the
-    agg template source statically, so it will not affect anything on
-    your system outside of matplotlib.
-
-`qhull` 2012.1
-    A library for computing Delaunay triangulations.
-
-`ttconv`
-   truetype font utility
+Dependencies
+------------
+
+Matplotlib requires a large number of dependencies:
+
+  * `Python <https://www.python.org/downloads/>`_ (>= 2.7 or >= 3.4)
+  * `numpy <http://www.numpy.org>`_ (>= |minimum_numpy_version|)
+  * `setuptools <https://setuptools.readthedocs.io/en/latest/>`__
+  * `dateutil` (>= 1.1)
+  * `pyparsing <https://pyparsing.wikispaces.com/>`__
+  * `libpng <http://www.libpng.org>`__ ( >= 1.2) 
+  * `pytz <http://pytz.sourceforge.net/>`__
+  * FreeType` (>= 2.3)
+  * `cycler <http://matplotlib.org/cycler/>`__ (>= 0.10.0)
+  * `six <https://pypi.python.org/pypi/six>`_
+  * `backports.functools_lru_cache <https://pypi.python.org/pypi/backports.functools_lru_cache>`_
+    (for Python 2.7 only)
+  * `subprocess32 <https://pypi.python.org/pypi/subprocess32/>`_ (for Python
+    2.7 only)
+
+Optionally, you can also install a number of packages to enables better user
+interface toolkit. See :ref:`what-is-a-backend` for more details on the
+optional Matplotlib backends and the capabilities they provide.
+
+  * :term:`tk` (>= 8.3, != 8.6.0 or 8.6.1): for the TkAgg backend;
+  * :term:`pyqt` (>= 4.4): for the Qt4Agg backend;
```

Comment:
well… we have another section where we list dependencies: it is mentioned there.